server/v2/store/server.go

```diff
@@ -0,0 +1,42 @@
+package store
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"cosmossdk.io/core/transaction"
+	"cosmossdk.io/log"
+	serverv2 "cosmossdk.io/server/v2"
+)
+
+type StoreComponent struct{}
+
+func New() serverv2.ServerComponent[transaction.Tx] {
+	return StoreComponent{}
+}
+
+func (s StoreComponent) Init(appI serverv2.AppI[transaction.Tx], v *viper.Viper, logger log.Logger) (serverv2.ServerComponent[transaction.Tx], error) {
+	return s, nil
+}
+
+func (s StoreComponent) Name() string {
+	return "store"
+}
+
+func (s StoreComponent) Start(ctx context.Context) error {
```

Comment:
this is part of the extension interface, so let's delete those (Start and Stop)

Reply:
hmm I think it will break `ServerComponent` interface

Reply:
You are right, forgot that commands are config were optional, not start & stop